docs/wiki/Contributing-Code.md

```diff
@@ -67,10 +67,18 @@ The Roslyn team regularly uses the [.NET code formatter tool](https://github.com
 - `/nounicode`: In general we follow this rule of not having unicode characters embedded in string literals. However there are a few cases where this is needed to verify compiler behavior hence this option is disabled for now. 
 - `/copyright`: The default copyright is MIT.  Roslyn is released under Apache2 hence we need to override this option. 
 
-### Visual Basic Conventions
+### Visual Basic Conventions and Rules
 
 For all of the C# guidelines which have analogs in Visual Basic, the team applies the spirit of the guideline to Visual Basic. Guidelines surrounding spacing, indentation, parameter names, and the use of named parameters are all generally applicable to Visual Basic. ‘Dim’ statements should also follow the guidelines for the use of ‘var’ in C#. Specific to Visual Basic, field names should begin with ‘m_’ or ‘_’. And the team prefers that all field declarations be placed at the beginning of a type definition. The Visual Studio members dropdown does not show fields in VB. Having them at the beginning of the type aids in navigation.
 
+IDE features should generally be made for both C# and VB.  The exceptions are:
+1. If the feature has no appropriate VB analog. For example 'patterns' are C#-only, so specific features around patterns do not generally need equivalent VB work.
+2. The feature is prohibitively expensive to also do for VB.  In this case, ask the team if it's acceptable to not do a VB version and a decision can be made.  In general though, doing the equivalent VB side ends up only being around 5% more effort, so it should normally be done.
```

Comment:
Would it possibly be less intimidating if this was rephrased to something like, "... doing the equivalent VB side usually ends up taking a significantly smaller amount of effort"? My concern is that giving a percentage (although approximate) could potentially discourage the average person if it ends up taking more effort than that for them. (Just a thought - I could totally overthinking things. :))

Reply:
>doing the equivalent VB side usually ends up taking a significantly smaller amount 

I don't think that's true. The sentence says it's "more effort" and I can confirm. But it should be small if the first side was written with VB in mind.

Reply:
Done!